packages/idyll-components/src/h1.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import GenerateHeaders from './generateHeaders';
+
+class H1 extends React.PureComponent {
+  render() {
+    const { children } = this.props;
+    return <GenerateHeaders size="1">{children}</GenerateHeaders>;
```

Comment:
We should update this so that other properties get passed into the generate headers, e.g. we want to support:

```
[H1 id:"my-custom-id" className:"my-custom-className"] Header Text [/H1]
```

Reply:
You can use `<GenerateHeaders size="1" {...props}>{children}</GenerateHeaders>` to pass through all of the props to the generator

Reply:
Done !!